docker-compose.yml

```diff
@@ -184,7 +184,7 @@ services:
       CONNECT_PLUGIN_PATH: "/usr/share/java,/connect-plugins"
       CONNECT_LOG4J_ROOT_LOGLEVEL: INFO
       CONNECT_LOG4J_LOGGERS: org.reflections=ERROR
-      CLASSPATH: /usr/share/java/monitoring-interceptors/monitoring-interceptors-5.5.0-SNAPSHOT.jar
+      CLASSPATH: /usr/share/java/monitoring-interceptors/monitoring-interceptors-5.5.x-latest.jar
```

Comment:
For JARs, shouldn't we follow the SNAPSHOT pattern?

Reply:
You're right, this was me doing a global sed, and it picked this up. Will fix.

Reply:
fixed in next commit.